### DISCUSSION TITLE
Could not detect to which game the provided path belongs(Star Rail)

### DISCUSSION THREAD

**Discussion by wp760836** [alexpado/genshin-music-unpacker#21] [Q&A] [answered]
I tried many times and it just posted:
![image](https://github.com/user-attachments/assets/cf0c7a4f-b1db-40f3-a4d6-b4431f85bd60)
Did I do something wrong?

**Comment by alexpado** [comment]
Hi !
Have you installed the game using the new HoyoPlay launcher or is it installed manually ?

**Reply by wp760836** [reply]
the new HoyoPlay launcher
![image](https://github.com/user-attachments/assets/f8db758e-3f10-41d2-a215-b1ff77775212)
![image](https://github.com/user-attachments/assets/1d4b0834-f996-487d-b35f-27a4079899d9)

**Reply by alexpado** [reply]
Ok I need to roll an update as the new launcher use a slightly different directory structure.

**Reply by alexpado** [reply, marked as answer]
Try with the [1.4.1 release](https://github.com/alexpado/hoyoverse-music-unpacker/releases/tag/1.4.1), you should be able to extract the game musics now

**Reply by wp760836** [reply]
Works very well! Thanks !!!!

> 尝试使用[1.4.1 版本](https://github.com/alexpado/hoyoverse-music-unpacker/releases/tag/1.4.1)，你现在应该能够提取游戏音乐
